gallery/src/index.css

```diff
@@ -0,0 +1,111 @@
+body {
+    font-family: Arial, Helvetica, sans-serif;
+    color: white;
+}
+
+.restricted-media {
+    height: 100%;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.connection-banner,
+.main-window,
+.info-box {
+    border: 1px solid #308274;
+    border-radius: 10px;
+    background: black;
+}
+
+.connection-banner {
+    padding: 0 10px;
+    max-width: 50%;
+    width: 50%;
+    margin: 10px 0;
+    text-align: center;
+    word-break: break-all;
+}
+
+.connection-banner > h3 {
+    margin: 5px;
+}
+
+.container {
+    display: flex;
+    justify-content: center;
+}
+
+.item {
+    border: 1px solid #308274;
+    border-radius: 5px;
+    display: flex;
+    flex-direction: column;
+    margin: 5px;
+}
+
+.item-name {
+    word-break: break-all;
+    padding: 10px;
+}
+
+.restricted-image {
+    max-height: 200px;
+    max-width: 150px;
+}
+.placeholder {
+    height: 200px;
+    width: 150px;
+    background: white;
+    color: black;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.main-window {
+    text-align: center;
+    display: flex;
+    justify-content: space-around;
+    padding: 10px;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+    flex-grow: 1;
+    margin-bottom: auto;
+    align-items: flex-start;
+    max-width: 50%;
+    align-self: center;
+}
```

Comment:
Would do this instead:

```suggestion
.main-window {
    text-align: center;
    display: grid;
    grid-template-columns: repeat(auto-fill, 150px);
    justify-content: center;
    padding: 10px;
    flex-grow: 1;
    margin-bottom: auto;
    max-width: 50%;
}
```

Reply:
Really?
![Screenshot from 2022-12-13 16-14-09](https://user-images.githubusercontent.com/87635671/207371589-97e0bc55-4693-413d-b285-ecbd5be6ded4.png)

Reply:
I guess changing it to 
grid-template-columns: repeat(auto-fill, 162px) (to account for 1px border and 5px of margin :thinking: ) 
made it nice.